models/post.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../db/connection')
+
+exports.insertCommentByArticleId = (article_id, username, body) => {
+    const created_at = new Date()
+
+    return db.query(`
+    INSERT INTO comments
+    (body, author, article_id, created_at)
+    VALUES
+    ($1, $2, $3, $4)
+    RETURNING author, body;`, [body, username, article_id, created_at] )
+    .then(({rows}) => {
+        return rows
```

Comment:
Better to return only the item here instead of the array so we've done all the work here in the model and pass the final data to the controller.